README.md

```diff
@@ -9,6 +9,7 @@ you'll need to configured that workflow yourself. You can look to the
 ### Workflow
 
 * Check for the latest version number published to npm.
+* If you wish privately publish your package please ensure you have set `NPM_PRIVATE` to `true`
```

Comment:
Go ahead and just add another section at the bottom for features and write up this feature along with an example. This is a great feature and I want to make sure people see the work you’ve done 😁